scipy/stats/tests/test_stats.py

```diff
@@ -4737,6 +4737,235 @@ def test_std_zero(self):
         assert_equal(stats.sigmaclip(x)[0], x)
 
 
+class TestAlexanderGovern(object):
```

Comment:
No need for `object` parent in Python 3
```suggestion
class TestAlexanderGovern:
```